static/themes/dark-theme.scss

```diff
@@ -142,6 +142,16 @@ a.navbar-brand img.logo.normal {
     color: #007bfd !important;
 }
 
+.flow-decoration {
+    background-color: #b4131f !important;
+    color: white !important;
+    font-weight: bold;
+}
+
+.flow-highlight {
+    background-color: rgba(180, 19, 31, 0.4) !important;
+}
```

Comment:
Are this colours visible indark mode? I noticed they are the same as the default light version

Reply:
Yes, I find it pretty good. WDYT?
![image](https://user-images.githubusercontent.com/95592999/179509067-1f83efd0-813b-494e-8589-02aae49b4522.png)